### ISSUE TITLE
Step 6  

### ISSUE THREAD

**Issue by grace-ac** [RobertsLab/Paper-DNR-Geoduck-Proteomics#18]
At the very end of your [first notebook](https://github.com/RobertsLab/Paper-DNR-Geoduck-Proteomics/blob/master/notebooks/SRM/01-Create-Skyline-Project.md#step-6-re-arrange-skyline-window) it says that my Skyline window should look like this:
![capture1](https://user-images.githubusercontent.com/14934314/32027265-f827191e-b99c-11e7-8954-73bbe313fd77.PNG)

However, mine looks like this:
![capture](https://user-images.githubusercontent.com/14934314/32027277-0041fba0-b99d-11e7-9fcc-d449086e21fb.PNG)

I selected the same peptide you did in your screenshot and the same tab (142), but there is nothing in the two side windows. 

these are the files that downloaded from the newest [RScript](https://raw.githubusercontent.com/RobertsLab/Paper-DNR-Geoduck-Proteomics/master/analyses/SRM/Script00-downloading-raw-files.R) that you have in your [00- notebook](https://github.com/RobertsLab/Paper-DNR-Geoduck-Proteomics/blob/master/notebooks/SRM/00-Download-raw-data.md).

**Comment by laurahspencer** [comment]
Not sure why nothing is visible on the Replicate Comparison panes on the right. Can you show me your Skyline document tomorrow (Thursday)? However ... 

... on my list of things to do is change file you use to populate the analyte tree from a fasta with all proteins to a .csv list of transitions. This will streamline the process of setting up a Skyline document. I'll try to get this done tonight and will update my notebooks to reflect the change.

**Comment by grace-ac** [comment]
Yes, I'll be in tomorrow and can show you what I've been working on!
